bedrock/firefox/templates/firefox/whatsnew-base-resp.html

```diff
@@ -0,0 +1,18 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-resp.html" %}
+
+{#- The firstrun and whatsnew pages should not be indexed by search engines -#}
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {% if l10n_has_tag('meta_update_201503') %}
+    {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
```

Comment:
I'm confused why we're now adding these to all whatsnew pages? As previously mentioned, these should be treated as internal FF only pages and are not really meant for sharing. Much of the code for many of these pages is only targeted for specific versions of Firefox.